app/models.py

```diff
@@ -257,7 +261,10 @@ def get_inbound_number(self):
 
     def get_default_sms_sender(self):
         default_sms_sender = [x for x in self.service_sms_senders if x.is_default]
-        return default_sms_sender[0].sms_sender
+        if default_sms_sender:
+            return default_sms_sender[0].sms_sender
+        else:
+            return 'gov'
```

Comment:
All services should have a default SMS sender. But if you want to check, then we should return GOVUK (our default value for new services)

Reply:
This was just some temporary code I wrote because my DB was messed up. I’ll squash it out.